### PR TITLE
Documented the "labels" option of DateIntervalType

### LINKED ISSUE #7368: [Form] Document the new labels of DateIntervalType
See https://github.com/symfony/symfony/pull/20887

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7368.

### CODE REVIEW

File: reference/forms/types/dateinterval.rst [hunk 1]
```diff
@@ -128,6 +129,26 @@ your underlying object. Valid values are:
...
+
+.. versionadded:: 3.3
+   The ``labels`` option was introduced in Symfony 3.3.
+
+**type**: ``array`` **default**: (see below)
+
+The labels displayed for each of the elements of this type. The default values are::
+
+  'labels' => array(
+      'invert' => 'form.trans.invert',
```
Comment (HeahDude) [inline reference/forms/types/dateinterval.rst]: Default option values are `null`, so default labels are humanized child names, so "Invert", "Years", ...

**Review by HeahDude — CHANGES_REQUESTED** [review]

File: reference/forms/types/dateinterval.rst (hunk 1 above)
Comment (javiereguiluz) [inline reference/forms/types/dateinterval.rst]: @HeahDude thanks for explaining the error. Hopefully I've fixed it in my other commit.

**Comment by HeahDude** [comment]
Thank you @javiereguiluz for documenting it!

👍

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**